chrome/content/zotero/xpcom/server_connector.js

```diff
@@ -594,6 +594,44 @@ Zotero.Server.Connector.Progress.prototype = {
 };
 
 /**
+ * Install a csl style or import a file using import translation
+ * 
+ * Accepts:
+ * 	[resourceURL, resourceType, resourceText]
```

Comment:
I think it'd be better to treat this (and other new endpoints) as somewhat more formal APIs rather than as internal hacks — that would make our own code clearer and also make it easier for others to build tools that use the same endpoints. We'd have to change things to make the `Content-Type` request header available to `init` (instead of using a `resourceType` string), and then we could use an object with clearer named properties for the other two params.